dds_cli/__main__.py

```diff
@@ -2071,21 +2071,21 @@ def deactivate_motd(click_ctx, motd_id):
 @motd_group_command.command(name="send")
 @click.argument("motd_id", metavar="[MOTD_ID]", nargs=1, type=int, required=True)
 @click.option(
-    "--unit-personnel-only",
+    "--unit-only",
     is_flag=True,
     required=False,
     default=False,
-    help="Send MOTD to unit personnel only.",
+    help="Send MOTD to unit personnel and unit admins only.",
```

Comment:
Since these are roles we should capitalize them, and I suggest rearranging the sentence for a better flow and with the highest hierarchy first.
```suggestion
    help="Only send MOTD to Unit Admins and Unit Personnel.",
```